app/src/main/java/org/thoughtcrime/securesms/conversation/v2/messages/VisibleMessageView.kt

```diff
@@ -75,7 +77,16 @@ class VisibleMessageView : LinearLayout {
     @Inject lateinit var mmsDb: MmsDatabase
     @Inject lateinit var lastSentTimestampCache: LastSentTimestampCache
 
-    private val binding by lazy { ViewVisibleMessageBinding.bind(this) }
+    private val binding = ViewVisibleMessageBinding.inflate(LayoutInflater.from(context), this, true)
+
+    private val markerContainerBinding = lazy(LazyThreadSafetyMode.NONE) {
```

Comment:
Why did you have to add the  LazyThreadSafetyMode.NONE to those two bindings?

Reply:
I find it wasteful to have a thread safe lazy for UI components where you only access through main thread.